### ISSUE TITLE
Dev: WAVE Accessibility: Create Project Pages, Form Labels

### ISSUE THREAD

**Issue by ryanfchase** [hackforla/tdm-calculator#2572]
### Overview

We need to fix an accessibility bug in the Create Project pages, to enable assistive technologies to associate form labels with their respective input elements.

#### WAVE Error: Orphaned Form Label, Missing Form Label
- [x] Download WAVE Tool Chrome Extension (see dropdown: Testing Resources)
  - [x] Run the WAVE Tool on Create Project Page 1
  - [x] Ensure you can see relevant WAVE Errors
    - see dropdown: Create Project, Page 1: Orphaned Form Label
    - see dropdown: Create Project, Page 1: Missing Form Label
- [x] In MultiInput.jsx, modify the render block for the `<Input>` Component (defined on L48)
  - [x] add id and name to the input element on L111
    - e.g. `<input {...} id={props.code} name={props.code} />`
  - [x] modify InputMask.id attribute, so that it does not clash with the the input.id      
    - from: `id={props.code}`
    - to: `id={'multi-input-' + props.code}`
- [x] Run the WAVE Tool on Create Project Page 1, confirm the errors are resolved (see dropdown: Create Project, Page 1: No Form Label Errors)
  - [x] Orphaned Form Label
  - [x] Missing Form Label

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, [2572](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3691501d13a_2_34#slide=id.g3691501d13a_2_34)]
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]

#### Developer Resources

<details><summary>Testing Resources</summary>
<p>

- Dev: https://tdm-dev.azurewebsites.net/calculation/1/0
- Prod: https://tdm.ladot.lacity.org/login
- [WAVE Chrome extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)

</p>
</details>

<details><summary>Definitions and Shorthand Terms</summary>
<p>

Term | Defined by WAVE | Definition
-- | -- | --
Form Label | Y | a `<label>` that is associated with a form control. In order to meet WAVE Tool criteria, the label.for  attribute must match the input.id attribute. Often referred to as a `<label>`, `<input>` pair.
Form Control | Y(?) | elements used in forms that allow users to input data, make selections, and interact with the form. In this ticket the form control is always an `<input>` element.
Orphaned Form Label | Y | a Form Label that the WAVE Tool cannot find its paired form control (`<input>`)
Missing Form Label | Y | a Form Control (`<input>`) that the WAVE Tool cannot find its paired `<label>`
Input Row | N | the container for a Form Label and its control, styled as a row. Can be found within the code as `<RuleInput />`, in `client/src/components/ProjectWizard/RuleInput/RuleInput.jsx`
APN | N | formally Assessor's Parcel Number, but here it describes the Form Label and control that triggers both the Orphaned Form Label error and the Missing Form Label WAVE error.
PROJECT_ADDRESS | N | an example Form Label that does not trigger any WAVE Tool errors. Referred to as the "good" example.

##### Visual Guide
![Image](https://github.com/user-attachments/assets/4b59b149-0101-4caa-9a07-6d6b8c812c81)

</p>
</details>

<details><summary>MultiInput.jsx, Input Component - Marked Up (copy/paste)</summary>
<p>

```jsx
// client/src/components/ProjectWizard/RuleInput/MultiInput.jsx

const Input = props => {
 // ...

  return (
    <div {/* ... */}>
      <InputMask
        {/* ... */}
        id={props.code} {/* we'll need modify this so that we can reserve props.code for input.id below */}
        {/* ... */}
      >
        {inputProps => (
          <input
            { /* need to add id and name so that it matches the "paired" label.for attribute (value will be props.code, which will be "APN") */ }
            {...inputProps}
            {/* ... */}
          >
        )}
      </InputMask>
    </div>
  );
};
```

</p>
</details> 

#### Screenshots

<details><summary>Create Project, Page 1: Orphaned Form Label</summary>
<p>

Description: A form label is present, but it is not correctly associated with a form control.

![Image](https://github.com/user-attachments/assets/a2c01d3d-ce13-4bfb-913a-f914f10f30e7)

</p>
</details>

<details><summary>Create Project, Page 1: Missing Form Label</summary>
<p>

Description: A form control does not have a corresponding label.

![Image](https://github.com/user-attachments/assets/89392973-31fb-40ba-885d-f52f81892d5e)

</p>
</details>

<details><summary>Create Project, Page 1: No Form Label Errors (For Verification)</summary>
<p>

After adjusting the code, we want to see that both errors are addressed:

1. We are no longer seeing the Missing Form Label error

![Image](https://github.com/user-attachments/assets/6b93e2ed-aa47-4f89-af55-33f014a084e1)

2. We are no longer seeing the Orphaned Form Label error

![Image](https://github.com/user-attachments/assets/0b611e8b-77e2-4ac5-a709-ad6c30680705)

</p>
</details>
